jobs/settings.py

```diff
@@ -1,11 +1,22 @@
 import os
 from datetime import timedelta
 
+
+# Force to cast strings to bool
+def str2bool(value: str, default: bool = False) -> bool:
```

Comment:
Hi, there is a function in the stdlib that does this convertion, `strtobool()`. It's checks for all this four 
possibilities and even for capital "T" of "F".
We would end up with something like:
 ```python
from distutils.util import strtobool

DEBUG = strtobool(os.environ.get('APF_DEBUG', False))  # returns 0 or 1
```
Does the same but a bit cleaner

Reply:
Nice fullonic! 

I'm gonna create a new ticket to improve it.

Thanks!

Reply:
Hi @fullonic, 

you can take this issue #42 and send me a pull request.

Thank you!

Reply:
Great. I would make the change and create the PR. Thanks :)